src/Components/v2/CarouselV2.tsx

```diff
@@ -220,7 +220,7 @@ export const LargeCarousel = (props: Props) => {
 export const SmallCarousel = (props: Props) => {
   const options = {
     draggable: true,
-    freeScroll: false,
+    freeScroll: true,
```

Comment:
Many images are wider than the mobile viewport so having `freescroll` enabled is definitely a better UX than having images lock when scrolled.